invenio_rdm_records/vocabularies/__init__.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+# Copyright (C) 2019 Northwestern University.
```

Comment:
Normal: 2020 😅 happens to me all the time too...

Reply:
Haha thanks! I had changed my new file template, but this file wasn't new, it was just the renamed models.py file. :sweat_smile: